extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -262,16 +270,37 @@ export class GoogleSubscriptionsPlatform {
     /** @const @private {!JsonObject} */
     this.serviceConfig_ = platformConfig;
 
+    /** @private viewer */
+    this.viewerPromise_ = Services.viewerForDoc(ampdoc);
+
     /** @private {boolean} */
     this.isGoogleViewer_ = false;
-    this.resolveGoogleViewer_(Services.viewerForDoc(ampdoc));
+    this.resolveGoogleViewer_(this.viewerPromise_);
 
     /** @private {boolean} */
     this.isReadyToPay_ = false;
 
     // Install styles.
     installStylesForDoc(ampdoc, CSS, () => {}, false, TAG);
 
+    /** @private @const {boolean} */
+    this.enableMetering_ = !!this.serviceConfig_['enableMetering'] || false;
```

Comment:
I think the `|| false` can be removed on lines 287 and 290. The `!!` boolean cast should be enough